test/runtests.jl

```diff
@@ -519,7 +519,7 @@ for (op, tr, shifted_op) ∈ zip((:NormL2,), (:NormLinf,), (:ShiftedGroupNormL2B
     @test φ(zeros(n)) == h(y + s)
     t = rand(n)
     t .*= ψ.Δ / ψ.χ(t) / 2
-    @test φ(t) == h(y + s + t)  # y inside the trust region
+    @test φ(t) ≈ h(y + s + t)  # y inside the trust region
```

Comment:
I think something is different with the way I implemented it for `ShiftedIndBallL0BInf` and `ShiftedGroupNormL2Binf` but the result is almost the same (difference close to `eps()`)